src/components/slider/slider.tsx

```diff
@@ -150,6 +151,21 @@ export class Slider implements LabelableComponent, FormComponent, InteractiveCom
    */
   @Prop() scale: Scale = "m";
 
+  /**
+   * When true, number values are displayed with the locale's group separator.
+   */
+  @Prop({ reflect: true }) groupSeparator = false;
+
+  /** Specifies the BCP 47 language tag for the desired language and country format. */
+  @Prop() locale: string = document.documentElement.lang || "en";
```

Comment:
These might be related to this:

https://github.com/Esri/calcite-components/issues/4576
https://github.com/Esri/calcite-components/issues/4213

Should we start prioritizing these? Ideally, we should just use the global `lang` attribute.

Reply:
Ok, I'll make the change to `lang` instead.

Reply:
These are scheduled for this month according to the roadmap. 😄

Reply:
In this PR's context, let's move forward w/ `lang`. I forgot `pagination` already was using our global util, so let's use that moving forward and we'll align other components this month.